xbmc/pvr/dialogs/GUIDialogPVRChannelGuide.h

```diff
@@ -46,5 +47,8 @@ namespace PVR
 
     CFileItemList    *m_vecItems;
```

Comment:
While you are at refactoring this, would it make sense to use a std::unique_ptr instead of a raw pointer here?